pkg/config/validation.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+type Validation struct {
+	DisabledValidators Validators `yaml:"disabled-validators,omitempty" json:"disabled-validators,omitempty"`
+	// ValidateConcurrently DO NOT USE THIS INTERNALLY! It is a user supplied setting and is migrated to Concurrent.
+	ValidateConcurrently *bool `yaml:"validate-concurrently,omitempty" json:"validate-concurrently,omitempty"`
```

Comment:
Can you simplify this?!
Simply call it `DisableConcurrency` and use a bool. It will default to `false` again (zero value of a bool) and we're good, also without the `validateSetDefaults` function which can also be removed then.